echopype/tests/convert/test_convert_ek80.py

```diff
@@ -169,17 +173,24 @@ def test_convert_ek80_cw_power_angle_echoview(ek80_path):
     # Convert file
     echodata = open_raw(ek80_raw_path_cw, sonar_model='EK80')
 
+    # get indices of sorted frequency_nominal values. This is necessary
+    # because the frequency_nominal values are not always in ascending order.
+    sorted_freq_ind = np.argsort(echodata.beam.frequency_nominal)
+
+    # get sorted channel list based on frequency_nominal values
+    channel_list = echodata.beam.channel[sorted_freq_ind.values]
+
     # check water_level
     assert (echodata["Platform"]["water_level"] == 0).all()
 
     # Test power
     # single point error in original raw data. Read as -2000 by echopype and -999 by EchoView
-    echodata.beam.backscatter_r[3, 4, 13174] = -999
-    for file, freq in zip(ek80_echoview_power_csv, freq_list):
+    echodata.beam.backscatter_r[sorted_freq_ind.values[3], 4, 13174] = -999
+    for file, chan in zip(ek80_echoview_power_csv, channel_list):
         test_power = pd.read_csv(file, delimiter=';').iloc[:, 13:].values
         assert np.allclose(
             test_power,
-            echodata.beam.backscatter_r.sel(frequency=freq * 1e3,
+            echodata.beam.backscatter_r.sel(channel=chan,
                                             beam='1').dropna('range_sample'),
```

Comment:
Given what you have below for comparing with `df_angle`, I suggest we have the same thing here (i.e., to index based on `chan` in `channel_list`) for this comparison of power. The echoview output should be in the same frequency/channel order for angle and power.

Reply:
> Given what you have below for comparing with df_angle, I suggest we have the same thing here

@leewujung I am a little confused at what you are suggesting here. Line 193 seems to be the same way of indexing. Maybe you are referring to line 188 above?

Reply:
Sorry I think I have misread the code somehow!